docs/docs/scanner/misconfiguration/index.md

```diff
@@ -326,13 +326,13 @@ trivy config --misconfig-scanners=terraform,dockerfile .
 
 Will only scan for misconfigurations that pertain to Terraform and Dockerfiles.
 
-### Passing custom checks
-You can pass policy files or directories including your custom checks through `--policy` option.
+### Loading custom checks
+You can load policy files or directories including your custom checks through `--config-check` flag.
```

Comment:
For some reason, GitHub has seemed to disable suggestions for me. But in this line it should be `load check files` instead of policy.